hack/generate-crd.sh

```diff
@@ -29,7 +29,7 @@ echo "---------------------------------------------------------------"
 oc kustomize "${MANIFESTS_DIR}" | \
     awk -v clf="${MANIFESTS_DIR}/${CLF_CRD_FILE}" \
         -v clo="${MANIFESTS_DIR}/${CLO_CRD_FILE}"\
-        'BEGIN{filename = clf} /---/ {getline; filename = clo}{print $0> filename}'
+        'BEGIN{filename = clf} /---/ {getline; filename = clo}{print $0> filename}' || { echo 'oc kustomize command failed'; exit 1; }
```

Comment:
Why using this ugly command and not `set pipefail`?

Reply:
Fixed